c/CMakeLists.txt

```diff
@@ -1,4 +1,4 @@
-cmake_minimum_required(VERSION 3.2)
+cmake_minimum_required(VERSION 3.12)
```

Comment:
Needed to up this number because [here](https://cmake.org/cmake/help/latest/module/GenerateExportHeader.html) you can see that this feature only worked for C++ until until 3.12 when C was also introduce.